cerberus/cerberus.py

```diff
@@ -758,6 +758,17 @@ def _validate_valueschema(self, schema, field, value):
                 if len(validator.errors):
                     self._error(field, validator.errors)
 
+    def _validate_excludes(self, exclude, field, value):
+        targets = []
+        if isinstance(exclude, list):
```

Comment:
please test for `isinstance(exclude, Sequence) and not isinstance(exclude, _str_types)`